sockeye/inference.py

```diff
@@ -813,6 +841,12 @@ def _concat_translations(translations: List[Translation], start_id: int, stop_id
             else:
                 target_ids.extend(translation.target_ids[1:])
                 attention_matrices.append(translation.attention_matrix[1:, :])
+        if translation.beam_history:
+            # Make a list of the individual beam histories
+            beam_histories.append(translation.beam_history[0])
+
+    if not beam_histories:
```

Comment:
Could we go forward with an empty list here? These two lines look a bit confusing at first.